src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java

```diff
@@ -32,12 +26,19 @@ public void execute() throws MojoExecutionException {
 
         String applicationId = getApplicationId(contrast, appName);
 
-        long serverId = getServerId(contrast, applicationId);
+        List<Long> serverIds = null;
+
+        if (StringUtils.isNotBlank(serverName)) {
+            serverIds = getServerId(contrast, applicationId);
+        }
 
         TraceFilterForm form = new TraceFilterForm();
         form.setSeverities(getSeverityList(minSeverity));
         form.setAppVersionTags(Collections.singletonList(computedAppVersion));
-        form.setServerIds(Arrays.asList(serverId));
+
+        if (serverIds != null) {
```

Comment:
@serge17 Looking good. Can you stand up a test for this to make sure we're filling out the filterForm correctly? I don't think we have tests around the Verify yet.

Reply:
@gmoore A test class added.